modules/tinymce/src/core/main/ts/api/EditorOptions.ts

```diff
@@ -187,7 +196,7 @@ const processDefaultValue = <T, U>(name: string, defaultValue: T, processor: Pro
 
 const create = (editor: Editor, initialOptions: Record<string, unknown>): Options => {
   const registry: Record<string, OptionSpec<any, any>> = {};
-  const values: Record<string, any> = {};
+  const values: Record<string, { value: any; isDefault: boolean }> = {};
```

Comment:
I'm guessing this needs rebasing against the other branch?

Reply:
Yeah it does... I haven't done that yet sorry as I only merged it about an hour ago.

Reply:
This has been rebased now.